backend/src/relay.ts

```diff
@@ -137,6 +137,12 @@ async function *fetchBlocksInBatches(
     // TODO: Cache of mapping from block number to its hash for faster fetching
     const [blockHash, block] = await pRetry(
       () => getBlockByNumber(httpUrl, nextBlockToProcess),
+      {
+        forever: true,
+        minTimeout: 1000,
+        maxTimeout: 2000,
```

Comment:
2 seconds? That is WAY TOO LOW, should be like 30 minutes or an hour, otherwise your log will be full of useless messages and DataDog price tag will be ridiculous. Not to say that it will defeat the purpose of having exponential backoff.